 AudioFlinger: Increase shared heap size to 8MB

For consecutive 24bit, 192kHz, 7.1 content 4MB heap is not enough.
So increasing heap size to 8Mb to avoid audio playback failure
for high resolution multi channel content.

diff --git a/services/audioflinger/AudioFlinger.h b/services/audioflinger/AudioFlinger.h
index 64c7edc..9008078 100644
--- a/services/audioflinger/AudioFlinger.h
+++ b/services/audioflinger/AudioFlinger.h
@@ -92,7 +92,7 @@ static const nsecs_t kDefaultStandbyTimeInNsecs = seconds(3);
 // Max shared memory size for audio tracks and audio records per client process
 static const size_t kClientSharedHeapSizeBytes = 1024*1024;
 // Shared memory size multiplier for non low ram devices
-static const size_t kClientSharedHeapSizeMultiplier = 4;
+static const size_t kClientSharedHeapSizeMultiplier = 8;
 
 #define INCLUDING_FROM_AUDIOFLINGER_H
 
